Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -1743,24 +1742,36 @@ public virtual float BaseFontSize
             }
         }
 
-        [AllowNull, DefaultValue(typeof(Font), DEFAULT_FONT_STYLE)]
+        /// <summary>Gets or sets the base palette font.</summary>
+        /// <value>The base palette font.</value>
+        [AllowNull, Description(@"Gets or sets the base palette font."), DefaultValue(null)]
         public virtual Font? BaseFont
         {
-            get => _baseFont ?? (Font)new FontConverter().ConvertFromString(DEFAULT_FONT_STYLE);
+            get => _baseFont ?? DEFAULT_FONT_STYLE;
 
             set
             {
-                _baseFont = value;
-
-                if (value == null)
+                if (value != null)
                 {
-                    _baseFont = (Font)new FontConverter().ConvertFromString(DEFAULT_FONT_STYLE);
+                    _baseFont = value;
+
+                    // Note: Might be overkill
+                    if (value == null)
+                    {
+                        _baseFont = DEFAULT_FONT_STYLE;
+                    }
                 }
             }
         }
 
+        /// <summary>Gets or sets the name of the theme.</summary>
+        /// <value>The name of the theme.</value>
+        [Description(@"Gets or sets the name of the theme.")]
         public virtual string? ThemeName { get => _themeName; set => _themeName = value; }
```

Comment:
This needs to be `DisallowNull` so that the code is cleaner.

Reply:
Usage of `DisallowNull`